src/lib.rs

```diff
@@ -127,7 +127,7 @@ A: Sized + ApplicationBase<gfx_device_gl::Resources, gfx_device_gl::CommandBuffe
     let mut events_loop = glutin::EventsLoop::new();
     let (window, mut device, mut factory, main_color, main_depth) =
         gfx_window_glutin::init::<ColorFormat, DepthFormat>(window, context, &events_loop);
-    let (mut cur_width, mut cur_height) = window.get_inner_size_points().unwrap();
+    let (mut cur_width, mut cur_height) = window.get_inner_size().unwrap();
```

Comment:
oh my, this inner size story is sad. First, this method was [deprecated](https://docs.rs/winit/0.8.0/winit/struct.Window.html#method.get_inner_size) in favor of the new ones. Now the new ones are deprecated...

Reply:
Yep, winit is really at the hurting edge of multiplatform code though so I cut them more slack. Tbf in this case the gfx example code is pretty much unaffected.